components/create_post/index.js

```diff
@@ -46,6 +48,7 @@ function mapStateToProps() {
         const getCommentCountForPost = makeGetCommentCountForPost();
         const latestReplyablePostId = getLatestReplyablePostId(state);
         const currentChannelMembersCount = getCurrentChannelStats(state) ? getCurrentChannelStats(state).member_count : 1;
+        const tutorialStep = parseInt(get(state, Preferences.TUTORIAL_STEP, getCurrentUserId(state), TutorialSteps.FINISHED), 10);
```

Comment:
The old preference store has a `getPreferenceInt` but the new redux ones only have a boolean equivalent. I opted to converge on the simple `parseInt` and have a separate discussion about more flexible accessors from exported redux store methods.